docs/source/user_guide/config.rst

```diff
@@ -24,6 +24,10 @@ FiftyOne supports the configuration options described below:
 |                               |                                     |                               | specifying a custom MongoDB database to which to connect. See                          |
 |                               |                                     |                               | :ref:`this section <configuring-mongodb-connection>` for more information.             |
 +-------------------------------+-------------------------------------+-------------------------------+----------------------------------------------------------------------------------------+
+| `timezone`                    | `FIFTYONE_TIMEZONE`                 | `None`                        | An optional timzone string. If provided, all datetimes read from FiftyOne datasets     |
```

Comment:
Should the default be `local`?

Reply:
My thinking was that timezones are pretty error prone, especially for teams working across different timezones, and defaulting to UTC was my way of suggesting to the world that UTC is a good convention to use